assets/js/modules/analytics/setup.js

```diff
@@ -887,42 +883,38 @@ class AnalyticsSetup extends Component {
 			return null;
 		}
 
-		let showError = true; // default error message.
-		let showNotice = false;
+		let showErrorFormat = true; // default error message.
 		let message = errorMsg;
 
 		switch ( true ) {
 				case 'google_analytics_existing_tag_permission' === errorCode:
-					showError = false;
-					showNotice = true;
+					showErrorFormat = false;
 					break;
 				case onSettingsPage && errorCode && 'insufficientPermissions' === errorReason:
-					showError = false;
-					showNotice = true;
+					showErrorFormat = false;
 					message = __( 'You currently don\'t have access to this account.You can either request access from your team, or remove this Analytics tag and connect to a different account.', 'google-site-kit' );
 					break;
 				case ! onSettingsPage && 0 === accounts.length:
-					showError = false;
-					showNotice = true;
+					showErrorFormat = false;
 					message = __( 'Looks like you don\'t have Analytics account yet. Once you create it click "Re-fetch my account" and Site Kit will locate it.', 'google-site-kit' );
```

Comment:
I know it's unrelated but the sentence feels wrong. Shouldn't it be more like this?

```suggestion
					message = __( 'Looks like you don\'t have an Analytics account yet. Once you create it, click on "Re-fetch my account" and Site Kit will locate it.', 'google-site-kit' );
```